deepchecks/utils/wandb_utils.py

```diff
@@ -16,6 +16,9 @@
 __all__ = ['wandb_run']
 
 
+WANDB_INSTALLATION_CMD = 'pip install "wandb>=0.12.15" "protobuf>=3.12.0,<4.0dev"'
```

Comment:
why we need a specific version? latest is enough

Reply:
`"wandb>=0.12.15"` - this I copied from the dev-requirements
 "protobuf>=3.12.0,<4.0dev" - this is needed because otherwise wandb will raise a `ImportError`

Reply:
no need for the user to have a specific wandb, would also look weird.
is the thing with protobuf a known issue? seems weird it is needed

Reply:
> no need for the user to have a specific wandb, would also look weird.

it is not a specific wandb 
it is any wandb that is greater than 0.12

> is the thing with protobuf a known issue? seems weird it is needed

this problem appeared today in the morning CI/CD checks were failing
because somewhy `protobuf` greater than 4.0 were installed

Reply:
hm, indeed if deepchecks is installed from the python package index this problem with protobuf is not present, probably something is in our `dev-requirements` that installs protobuf>4.0

fixed